### DISCUSSION TITLE
How to enable overlay when fixed is turned off?

### DISCUSSION THREAD

**Discussion by aurariddle** [gethinode/hinode#1118] [Help] [answered]
For [navigation] in params.toml, when I turn on overlay and turn off fixed, the overlay effect stops working (i.e., the navigation bar is no more transparent). Without keeping the navigation bar fixed at the top of the screen, how can I make it transparent to match the color of the homepage?

**Comment by markdumay** [comment, marked as answer]
I've included a fix in https://github.com/gethinode/hinode/releases/tag/v0.26.4. The navbar should now support overlay irrespective of being fixed to the top. You may need to clear your browser cache for the themes to work correctly. Feel free to share your experience / feedback.
